.appveyor.yml

```diff
@@ -0,0 +1,64 @@
+# AppVeyor.com is a Continuous Integration service to build and run tests under
+# Windows
+
+environment:
+  REPO_DIR: scikit-image
+  WHEELHOUSE_UPLOADER_USERNAME: appveyor-worker
+  # https://www.appveyor.com/docs/build-configuration/#secure-variables
+  WHEELHOUSE_UPLOADER_SECRET:
+    secure: TBD
+  matrix:
+    - PYTHON: C:\Python27
+    - PYTHON: C:\Python27-x64
+    - PYTHON: C:\Python35
+    - PYTHON: C:\Python35-x64
+    - PYTHON: C:\Python36
+    - PYTHON: C:\Python36-x64
+
+matrix:
+  fast_finish: true
+
+install:
+  - ECHO "Filesystem root:"
+  - ps: "ls \"C:/\""
+
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "python -m pip install -U pip"
+
+  # Check that we have the expected version and architecture for Python
+  - "python --version"
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  - "pip --version"
+
+  # Get stdint headers needed by tifffile.c.
+  - "curl https://raw.githubusercontent.com/chemeris/msinttypes/master/inttypes.h -o skimage/external/tifffile/inttypes.h"
```

Comment:
The scikit-image repository is not at the root of this repo, but in a directory called`scikit-image`...

Reply:
Yes, line 40 cd's to this directory before operating on the repo.